internal/tsoptions/enummaps.go

```diff
@@ -217,6 +217,11 @@ var targetToLibMap = map[core.ScriptTarget]string{
 	core.ScriptTargetES2015: "lib.es6.d.ts", // We don't use lib.es2015.full.d.ts due to breaking change.
 }
 
+// Used for test to add the files
```

Comment:
I would just delete this; I initially read this as it saying that a test was _modifying_ the map, when it's just adding fake files.

Reply:
That being said, why do we need this when the harness already knows how to do this without this mapping?

Reply:
Harness usually parsed tsconfig so it has info on which default lib is needed.. also all the libs are bundled and available.i want to fake presence of all libs with the content so that the output is determinitstic and not giant for tsc tests